src/components/Artists/ArtistsList/ArtistList.js

```diff
@@ -0,0 +1,30 @@
+import { ArtistItemComponent } from "../ArtistsItem/ArtistItem";
+import ArtistListTemplate from "./ArtistList.html";
```

Comment:
template names should start from lowecase

Reply:
Fixed